content/docs/guide-development.md

```diff
@@ -62,7 +62,67 @@ For example:
 source_name = "Crab" # oda:AstrophysicalObject
 reference_energy = 20 # oda:keV
 ```   
- ### How to annotate the notebook outputs
+
+### Adding an external resource annotations
+
+In case your notebook relies on some external resources, such as S3 storage or compute cluster this should be reflected in the annotations in the notebook cell tagged `parameters`. Below is the list of the resource annotations supported currently:
```

Comment:
```suggestion
In case your notebook explicitly calls some external resources, such as S3 storage or compute cluster this should be reflected in the annotations in the notebook cell tagged `parameters`. Below is the list of the resource annotations supported currently:
```